internal/pubsub/mempubsub/conformance_test.go

```diff
@@ -23,38 +23,29 @@ import (
 	"gocloud.dev/internal/pubsub/drivertest"
 )
 
-type harness struct {
-	b *Broker
-}
+type harness struct{}
 
 func newHarness(ctx context.Context, t *testing.T) (drivertest.Harness, error) {
-	return &harness{b: NewBroker([]string{"t"})}, nil
+	return &harness{}, nil
 }
 
 func (h *harness) MakeTopic(ctx context.Context) (driver.Topic, error) {
-	dt := h.b.topic("t")
-	return dt, nil
+	return &topic{}, nil
 }
 
 func (h *harness) MakeNonexistentTopic(ctx context.Context) (driver.Topic, error) {
-	dt := h.b.topic("nonexistent-topic")
-	return dt, nil
+	return (*topic)(nil), nil
```

Comment:
Might be worth commenting here that a nil `*topic` is handled correctly as a nonexistent topic, it's not obvious.